src/style.css

```diff
@@ -1,3 +1,71 @@
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100vw;
+  background-color: white;
+}
+
 body {
   margin: 0;
 }
+
+.img_link {
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  height: 60px;
+}
+
+.nav__list {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0 50px 0 0;
```

Comment:
```suggestion
  margin-right: 50px;
```

Reply:
but as i've written above better to set padding on header, it can be removed then

Reply:
actually it's important, it's why you can't pass tests
![image](https://github.com/mate-academy/layout_moyo-header/assets/55200530/2760a495-48d8-4b96-adcf-4df0dc151dec)